src/core/server/services/comments/moderation/index.ts

```diff
@@ -111,6 +112,9 @@ const moderate = (
     result.comment.id,
     input.moderatorID
   );
+  if (result.oldStatus === "PREMOD" && status === "APPROVED") {
```

Comment:
I'd check against any unpublished status instead, as `SYSTEM_WITHHELD` could move to `APPROVED`.

Reply:
i was worried about getting double-notified in the case of a stream that is not premod but I guess that wouldn't happen?